client/Assets/Scripts/Battle.cs

```diff
@@ -102,6 +101,12 @@ void Update()
             SetAccumulatedTime();
             UpdateBattleState();
         }
+
+        if (LobbyConnection.Instance.gameStarted && !sendMovementStarted) {
+            sendMovementStarted = true;
+            float clientActionRate = SocketConnectionManager.Instance.serverTickRate_ms / 1000f;
+            InvokeRepeating("SendPlayerMovement", clientActionRate, clientActionRate);
```

Comment:
```suggestion
            InvokeRepeating("SendPlayerMovement", 0, clientActionRate);
```

It's not necessary to wait the first time